google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/SpannerTableExporter.java

```diff
@@ -82,4 +88,19 @@ public void initializeTableExporter(Metadata metadata, Action schemaAction) {
     return ddlStatements.toArray(new String[ddlStatements.size()]);
   }
 
+  /**
+   * Processes the columns of the table and creates Unique Constraints for columns
+   * annotated with @Column(unique = true).
+   */
+  private static void initializeUniqueConstraints(Table table) {
+    Iterator<Column> colIterator = table.getColumnIterator();
+    while (colIterator.hasNext()) {
+      Column col = colIterator.next();
+      if (col.isUnique()) {
+        String name = Constraint.generateName("UK_", table, col);
```

Comment:
Since "UK_" is used twice, consider making a constant. As a side benefit, I'll stop thinking about United Kingdom :D

Reply:
After removing the previous usage it will be used once so i'll just leave it inline.